### ISSUE TITLE
5-22-NFS-Volume-Gather NFS logs fails with permission denied

### ISSUE THREAD

**Issue by rajanashok** [vmware/vic#7605]
```
TEARDOWN Gather NFS Logs
Start / End / Elapsed:	20180329 06:45:26.209 / 20180329 06:45:30.332 / 00:00:04.123
00:00:02.078KEYWORD ${out} = BuiltIn . Run Keyword And Continue On Failure Run, sshpass -p %{DEPLOYED_PASSWORD} ssh -o StrictHostKeyChecking\=no root@${NFS_IP} dmesg -T
Documentation:	
Runs the keyword and continues execution even if a failure occurs.

Start / End / Elapsed:	20180329 06:45:26.210 / 20180329 06:45:28.288 / 00:00:02.078
00:00:02.076KEYWORD OperatingSystem . Run sshpass -p %{DEPLOYED_PASSWORD} ssh -o StrictHostKeyChecking\=no root@${NFS_IP} dmesg -T
Documentation:	
Runs the given command in the system and returns the output.

Start / End / Elapsed:	20180329 06:45:26.211 / 20180329 06:45:28.287 / 00:00:02.076
06:45:26.214	INFO	Running command 'sshpass -p 'ca\$hc0w' ssh -o StrictHostKeyChecking=no root@10.162.60.230 dmesg -T 2>&1'.	
06:45:28.287	INFO	${out} = Permission denied, please try again.	
00:00:00.001KEYWORD BuiltIn . Log ${out}
Documentation:	
Logs the given message with the given level.

Start / End / Elapsed:	20180329 06:45:28.289 / 20180329 06:45:28.290 / 00:00:00.001
06:45:28.289	INFO	Permission denied, please try again.
```

**Comment by rajanashok** [comment]
https://storage.cloud.google.com/vic-ci-logs/vic_nightly_logs_2018-03-29-04-41-17.zip?authuser=1

**Comment by rajanashok** [comment]
Fixed by #7646 
Authentication failed for user root
https://github.com/vmware/vic/issues/7641
